lib/libutee/include/pta_system.h

```diff
@@ -182,4 +182,14 @@
  */
 #define PTA_SYSTEM_DLSYM                11
 
+/*
+ * Retrieves a copy of the TPM Event log held in secure memory.
+ *
+ * [out]	memref[0].buffer: Pointer to a buffer where to store the event log.
```

Comment:
It's enough to just describe what's passed in `memref[0]`. The buffer pointer and size has the normal out buffer usage, compare with `PTA_SYSTEM_COPY_FROM_TA_BINARY` above.